includes/class-sensei-grading.php

```diff
@@ -470,20 +470,18 @@ public function courses_drop_down_html( $selected_course_id = 0 ) {
 	 * @return string
 	 */
 	public function get_lessons_dropdown() {
+		if ( ! isset( $_GET['course_id'] ) ) {
```

Comment:
This will also break backwards compatibility. Could we perhaps keep the POST request (even though semantically incorrect) and do a nonce check instead?

Reply:
I've added a fallback for this. Testing instructions from description:

> To test for backwards compatibility, copy the file `assets/js/grading-general.min.js` from the `master` branch into this branch and test again. Everything should work, and you should see some "Doing it wrong" messages in the error log.

Note that I did not add a nonce check. It is not needed (because there are no modifications), and technically it would also be a backwards compatibility break since a nonce isn't being supplied (`woo_localized_data.get_lessons_dropdown_nonce` doesn't actually exist on the page).